test/wdio.conf.mjs

```diff
@@ -281,37 +281,27 @@ function buildConfig() {
       },
 
       // iOS
-      // iOS11 currently failing to start with 'COULD NOT START MOBILE BROWSER'.
-      // https://automate.browserstack.com/dashboard/v2/builds/2840b7364373dc5f6f27be58d7bb33374d492668?overallStatus=error
-      // {
-      //   'bstack:options': {
-      //     'osVersion': '11',
-      //     'deviceName': 'iPhone 8',
-      //     'realMobile': 'true',
-      //   },
-      //   'browserName': 'iPhone',
-      // },
       {
         'browserName': 'iPhone',
         'bstack:options': {
-          'osVersion': '12',
+          'osVersion': '11',
```

Comment:
We run the integration tests against a variety of devices and browsers with Browserstack. Some of these configurations are no longer supported by Browserstack and were failing to run. I've updated the config based on [this list](https://www.browserstack.com/list-of-browsers-and-platforms/automate) of supported devices.